File: Sorting/Java/Selectionsort.java
package Sorting;

public class Selectionsort {

	
	public int[] selection(int[] arr){
		
		int i =0;
		while ( i < arr.length){
			int j = i + 1;
			while ( j < arr.length){
				if( arr[i] > arr[j]){
					int temp = arr[i];
					arr[i] = arr[j];
					arr[j] = temp;
				}
				j ++;
			}
		    i ++;
		}
		return arr;
		
	}
	
	public static void main(String s[]){
		Selectionsort  sort = new Selectionsort();
		int[] arr = {64, 25, 12, 22, 11};
		int[] ans = sort.selection(arr);
		
		System.out.println(ans[4]);
	}
}

File: Sorting/Python/selectionsort.py
def selectionsort(arr):
    i=0
    while i < len(arr):
        j = i+1
        while j < len(arr):
            if arr[i] > arr[j]:
                temp = arr[i]
                arr[i] = arr[j]
                arr[j] = temp
                
            j += 1
        i += 1
    return arr


print selectionsort([64, 25, 12, 22, 11])